client/js/upload.ts

```diff
@@ -116,6 +116,7 @@ class Uploader {
 
 		const wasQueueEmpty = this.fileQueue.length === 0;
 		const maxFileSize = store.state.serverConfiguration?.fileUploadMaxFileSize || 0;
+		const maxFileSizeInMB = (maxFileSize / 1024 / 1024).toFixed(0);
```

Comment:
This should probably be moved into the if guard... there's no need to calculate it when you don't hit the error.

Helps the reader too that the variable has a narrow scope and isn't used elsewhere.

Reply:
Good catch.